dataplug-google-calendar/app/org/hatdex/dataplugCalendar/apiInterfaces/GoogleCalendarEventsInterface.scala

```diff
@@ -64,7 +64,8 @@ class GoogleCalendarInterface @Inject() (
     hatClient: AuthenticatedHatClient,
     fetchParameters: ApiEndpointCall)(implicit ec: ExecutionContext, timeout: Timeout): Future[Done] = {
 
-    val validatedData = transformData(content, fetchParameters.pathParameters("calendarId")).map(validateMinDataStructure)
+    val validatedData = transformData(content, fetchParameters.pathParameters("calendarId"), fetchParameters.storageParameters.get("calendarName"))
```

Comment:
Unsafe (unfortunately, very implicit) unwrapping of Option, might throw an Exception.
More on this here: https://stackoverflow.com/questions/16202929/optionmapstring-string-can-get-a-value-weirdly